File: uri/1 - Basics/1020/Main.java
import java.util.*;

class Main{
	public static void main(String [] args){
	    Scanner scanner = new Scanner(System.in);
	    int a = scanner.nextInt();
	    System.out.println( a/365 + " ano(s)" );
	    System.out.println( (a%365)/30 + " mes(es)");
	    System.out.println( (a%365)%30 + " dia(s)");
	}
}

File: uri/1 - Basics/1020/Main.cs
using System;

class URI{
	public static void Main(string [] args){
	    int a = Int32.Parse( System.Console.ReadLine().Trim());
	    Console.Write( a/365 + " ano(s)\n" );
	    Console.Write( (a%365)/30 + " mes(es)\n");
	    Console.Write( (a%365)%30 + " dia(s)\n");
	}
}